Documentation/ceph-upgrade.md

```diff
@@ -321,7 +331,31 @@ At this point, your Rook operator should be running version `rook/ceph:v1.1.0`
 
 Verify the Ceph cluster's health using the [health verification section](#health-verification).
 
-### 6. (Recommended) Migrate config overrides from ConfigMap to Ceph directly
+### 6. Delete CSI provisioner and attacher (if applicable)
```

Comment:
Why is this step necessary? Doesn't the operator delete the old one automatically before creating the new one? It should.

Reply:
no operator is not doing this one.

Reply:
@BlaineEXE @travisn I asked the same question here https://github.com/rook/rook/pull/3642#issuecomment-524222684

Reply:
Fuck. I definitely missed the CSI upgrade case in my testing ...